lobby-db/README.md

```diff
@@ -0,0 +1,94 @@
+## lobby-db
+
+A project that contains the migration scripts for the lobby postgres database.
+
+### Prod Deployments
```

Comment:
Did you happen to try this on a copy of the production DB?  I'm curious how Flyway handles the fact that migration scripts 1.00-1.02 have already been applied, but there is currently no Flyway history table that records that information.  Will Flyway attempt to re-apply those scripts and just error out?

Reply:
Out of scope for this documentation, but to satisfy curiosity it will error out:
```
> Error occurred while executing flywayMigrate
  Found non-empty schema(s) "public" without schema history table! Use baseline() or set baselineOnMigrate to true to initialize the schema history table.
```

Reply:
Good to know.  So it would seem that the `flywayBaseline` task is going to need to be run against the production and staging databases before the `flywayMigrate` task.

Reply:
Exact commands TBD, but otherwise yes  : )